tsdb/block.go

```diff
@@ -112,7 +112,7 @@ type ChunkReader interface {
 // BlockReader provides reading access to a data block.
 type BlockReader interface {
 	// Index returns an IndexReader over the block's data.
```

Comment:
Probably needs an update. Note that there can be false positives.

Reply:
Which false positive?

Reply:
Firstly you're not implementing this for blocks, secondly that a range overlaps a min/max doesn't mean it has samples there.